Add blogpost for Kubewarden 1.27.3 release of kwctl (#363)
kubewarden/kubewarden.io: content/blog/2025/07/kubewarden-1.27.3-release.md

---
title: "Kubewarden 1.27.3 Patch Release"
authors:
  - Víctor Cuadrado Juan
date: 2025-08-13
---

We have just released 1.27.3, a small patch release for `kwctl`.
This newly released kwctl version `v1.27.3` fixes a bug on the
`kwctl run` subcommand for _ClusterPolicyGroups_ and _PolicyGroups_.

When evaluating policies and policy groups, both `kwctl` and `policy-server`
take care of running the policies in the correct execution mode that the
policies have defined via their
[metadata](https://docs.kubewarden.io/tutorials/writing-policies/metadata).

> This means that Kubewarden policies that are Wasm modules intended to run as
> WASI are run as such; the same for example for Rust policies compiled to run as
> WAPC, or [OPA
> policies](https://docs.kubewarden.io/tutorials/writing-policies/rego/open-policy-agent/build-and-run#run).

Reviewer: ```suggestion
This means that Kubewarden policies that are Wasm modules intended to run as
WASI are executed as such. The same applies, for example, to Rust policies compiled for
WAPC or to [OPA
policies](https://docs.kubewarden.io/tutorials/writing-policies/rego/open-policy-agent/build-and-run#run).
```

Up until `kwctl` 1.27.2, when doing `kwctl run` of a ClusterPolicyGroup or
PolicyGroup, `kwctl` wasn't extracting the execution mode from the metadata information
of the policies part of the group, but using the default execution mode.

> This meant that, if mingling different types of policies, it would fail for
> example with:

Reviewer: ```suggestion
This meant that when mixing different policies, it would fail, for
example, with:
```
